src/Elcodi/Bundle/CouponBundle/Resources/config/doctrine/Coupon.orm.yml

```diff
@@ -49,6 +49,12 @@ Elcodi\Component\Coupon\Entity\Coupon:
             column: minimum_purchase_amount
             type: integer
             nullable: true
+        stackable:
+            column: stackable
+            type: boolean
+            nullable: false
+            options:
+                default: false
```

Comment:
This default data should not be set here. Instead of that, please change the CouponFactory class.